python/vowpalwabbit/objects.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
```

Comment:
https://github.com/scikit-learn/scikit-learn/blob/master/COPYING

Yes, this change might bring noise and implications in terms of licensing. I would advise to figure this out before merging.